### ISSUE TITLE
Use navigation view back button instead of title bar back button

### ISSUE THREAD

**Issue by MarcAnt01** [File-New-Project/EarTrumpet#533]
![image](https://user-images.githubusercontent.com/43090263/88652073-9f220700-d0ca-11ea-8078-0efbebd58022.png)

https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview

**Comment by riverar** [comment]
![image](https://user-images.githubusercontent.com/475132/88652876-39109200-d080-11ea-83f8-8976f5a97827.png)

This?

**Comment by MarcAnt01** [comment]
Yes, that's the actual implementation. Here's an example of what MS is suggesting to use rn
![image](https://user-images.githubusercontent.com/43090263/88653115-f1175c80-d0cb-11ea-8c8d-0c5cea1cf790.png)

**Comment by GoldenTao** [comment]
As we aren't a UWP application we're not using NavigationView, it's our similar looking but alternate design for settings written in WPF.

**Comment by MarcAnt01** [comment]
All right, got it, but is there any way to make the back button look more "modern"?

**Comment by GoldenTao** [comment]
At the moment we're emulating the look of the Windows settings app, is that not "modern"?

**Comment by MarcAnt01** [comment]
For me it's not and I'm wondering why they haven't been changing it. New Microsoft UWPs don't have that obsolete back button, that they recommend to avoid in the documentation, and most of them have already been updated to ditch it. I understand that EarTrumpet is not a UWP app, but since the design is similar to a UWP app, it should follow this trend, too.

**Comment by riverar** [comment]
Thanks @MarcAnt01! As @GoldenTao mentioned we are intentionally emulating the look of the Windows settings app. We're not looking to change that (unless the Settings app itself changes).
